_1327/information_pages/tests.py

```diff
@@ -222,6 +242,36 @@ def test_last_author(self):
 		response = self.app.get(self.document.get_view_url(), user=self.user)
 		self.assertIn('<i>by</i> {}'.format(user2.username), response.body.decode('utf-8'))
 
+	def test_show_author_to(self):
+		author_str = '<i>by</i> {}'.format(self.user.username)
+
+		self.document.show_author_to = InformationDocument.SHOW_AUTHOR_TO_EVERYONE
+		with transaction.atomic(), revisions.create_revision():
+			self.document.save()
+			revisions.set_user(self.user)
+			revisions.set_comment('test version 2')
+		self.assertIn(author_str, self.app.get(self.document.get_view_url(), user=self.user).body.decode('utf-8'))
+		self.assertIn(author_str, self.app.get(self.document.get_view_url(), user=self.unauthenticated_user).body.decode('utf-8'))
+
+		# The following part of the test is currently disables as the anonymous user's is_authenticated() method evaluates to true
```

Comment:
@Bartzi any ideas about the tests?

Reply:
you could do `context['request'].user.is_authenticated() and not context['request'].user.is_anonymous()` in [this](https://github.com/fsr-itse/1327/pull/649/files/fa16cf141f434cb38f37685c78741e1b0dc7aa70#diff-1b2e97b8158c25ff8530da9c362dc909R13) line

Reply:
maybe that helps